quickstart/exampledata/config/prometheus/prometheus.yml

```diff
@@ -29,7 +29,7 @@ scrape_configs:
       - targets: ["localhost:9090"]
   - job_name: "logprep"
     static_configs:
-      - targets: ["localhost:8000"]
+      - targets: ["localhost:8000", "localhost:8001"]
```

Comment:
Why add here another port? I didn't see that you added a second instance to the docker-compose file?

Reply:
this is if you want to start a second logprep on your machine.